styles/styles-grid.css

```diff
@@ -0,0 +1,426 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+:root {
+  --text-color: #383838;
+  --general-white: #ffffff;
+}
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  height: 100vh;
+}
+
+.svg-sprites {
+  position: absolute;
+  visibility: hidden;
+  width: 0;
+  height: 0;
+}
+
+.main {
+  background-image: url(../image/header/background-header.png);
+  background-repeat: no-repeat;
+}
+
+.container {
+  max-width: 1264px;
+  display: grid;
+  padding: 0 16px;
+  margin: 0 auto;
+}
+
+.lg-1-col {
+  grid-template-columns: 1fr;
+  grid-template-rows: auto;
+}
+
+.lg-2-col {
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: auto;
+}
+
+.lg-3-col {
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: auto;
+}
+
+.lg-4-col {
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: auto;
+}
+
+.lg-5-col {
+  grid-template-columns: repeat(5, 1fr);
+  grid-template-rows: auto;
+}
+
+.lg-6-col {
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-rows: auto;
+}
+
+.lg-7-col {
+  grid-template-columns: repeat(7, 1fr);
+  grid-template-rows: auto;
+}
+
+.lg-8-col {
+  grid-template-columns: repeat(8, 1fr);
+  grid-template-rows: auto;
+}
+
+.lg-9-col {
+  grid-template-columns: repeat(9, 1fr);
+  grid-template-rows: auto;
+}
+
+.lg-10-col {
+  grid-template-columns: repeat(10, 1fr);
+  grid-template-rows: auto;
+}
+
+.lg-11-col {
+  grid-template-columns: repeat(11, 1fr);
+  grid-template-rows: auto;
+}
+
+.lg-12-col {
+  grid-template-columns: repeat(12, 1fr);
+  grid-template-rows: auto;
+}
+
+.header {
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  grid-template-rows: auto;
+  grid-template-areas: "title . . nav-links nav-links mode login";
+  align-items: center;
+  justify-items: end;
+  gap: 24px;
+  padding-top: 50px;
+  margin-bottom: 120px;
+}
+
+.header--title {
+  grid-area: title;
+  width: 205px;
+  height: 40px;
+}
```

Comment:
It is better to align the logo to the left as it is moving with screen width changing now.

Reply:
I don't understand what you mean. Logo is aligned to the left edge all the time